test/parallel/test-inspector-workers-flat-list.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+
+const { Session } = require('inspector');
+const { Worker, isMainThread, parentPort } = require('worker_threads');
+
+const MAX_DEPTH = 3;
+
+let rootWorker = null;
+
+function runTest() {
+  let reportedWorkersCount = 0;
+  const session = new Session();
+  session.connect();
+  session.on('NodeWorker.attachedToWorker', ({ params: { workerInfo } }) => {
```

Comment:
ditto here, `common.mustCall(..., MAX_DEPTH)` for the event handler to make sure this actually runs as often as expected?

Reply:
Done